gizmo.py

```diff
@@ -0,0 +1,59 @@
+'''
+Created on Mar 31, 2022
+Gizmo Challenge File
+
+@author: thomas
+'''
+from spyder.app import start
+
+
+def hello(name,country='Finland'):
+    print("Hello {name}, how are things in {country}".format(name=name,country=country))
+    
+
+def spell(word):
+    spelled = ''
+    for i in range(len(word)-1):
+        spelled += word[i]+'.'
+    print(spelled + word[-1])
+
+def relative_path(subjects, basepath='./subjects/mock_recording_', extension='.rec' ):
+    return [basepath + subj + extension for subj in subjects]
```

Comment:
you were asked to use python's formatting syntax here:

```python
return [f'./subjects/mock_recording_{subj}.rec' for subj in subjects]
```

Also, designing `basepath` to be a combination of folder and part of a filename is a weird design choice.

Reply:
Should be updated